Jenkinsfile.release

```diff
@@ -13,40 +10,28 @@ pipeline {
         branch 'PR-*'
       }
       steps {
-        container('maven') {
-          sh "mvn install"
-        }
+        sh "mvn install"
       }
     }
     stage('Build Release') {
       when {
         branch 'master'
       }
       steps {
-        container('maven') {
-          // ensure we're not on a detached head
-          sh "git checkout master"
-          // until we switch to the new kubernetes / jenkins credential implementation use git credentials store
-          sh "git config --global credential.helper store"
-          sh "jx step git credentials"
+        // ensure we're not on a detached head
+        sh "git checkout master"
 
-          // so we can retrieve the version in later steps
-          sh "echo \$(jx-release-version) > VERSION"
-          sh "mvn versions:set -DnewVersion=\$(cat VERSION)"
+        // so we can retrieve the version in later steps
+        sh "echo \$(jx-release-version) > VERSION"
+        sh "mvn versions:set -DnewVersion=\$(cat VERSION)"
 
-          sh "git tag -fa v\$(cat VERSION) -m \"Release version \$(cat VERSION)\""
-          sh "git push origin v\$(cat VERSION)"
-        }
+        sh "git tag -fa v\$(cat VERSION) -m \"Release version \$(cat VERSION)\""
+        sh "git push origin v\$(cat VERSION)"
 
         container('maven') {
           sh 'mvn clean deploy -DaltDeploymentRepository=maven.jenkins-ci.org::default::https://repo.jenkins-ci.org/releases/ -P release'
```

Comment:
I do not think we really want to do this. Better to use the same `buildPlugin()` flow as other plugins.